### PR TITLE
our story page

### PR DESCRIPTION
**Pull request by teddymaulana**

### CODE REVIEW

**Review by foria — CHANGES_REQUESTED** [review]
DESKTOP
1. carousel is wrong, it should show 5 full elements without cutting them, check also the controls, let's just use the default desktop version, in this way we are consistent
2. the section at the bottom the text goes over the next image
3. there is no video icon and video popup, use the the code from Rizki about how-to blog section, adapting if needed, but don't create a custom code
4. for the hero-banner we should use hero-banner font-size that you can find the styleguide
5. the combination of text+image is something that we also use in the collection page, so let's create a compound and update both branches, in this we don't duplicate code

File: src/docs/compounds/image-with-text.hbs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			<div class="col-12 col-lg-8 order-lg-last">
+				<div class="no-gutters__in-container">
+					<picture>
+						<source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner04_DT_2x_6b055d51-178d-47e5-a6b1-c273f08a76aa_800x.jpg?v=1590061796" media="(min-width: 992px)" />
+						<img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner04_DT_2x_1000x_74c1ad4b-cb44-4123-a152-12d7495fbca4.jpg?v=1618404879" class="w-100">
+					</picture>
+				</div>
+			</div>
+			<figcaption class="col-12 col-lg-4">
+				<div class="image-with-text__text-content image-with-text__text-content--reverse text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pl-lg-3">
```
Comment (foria) [inline src/docs/compounds/image-with-text.hbs:38]: let's simply the naming calling just imagewtext, also reverse is a behaviour of the parent so the name should be imagewtext--reverse, the other just call it imagewtext__content

File: src/docs/templates/our-story.hbs [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="COMPONENT/COMPOUND/SECTION/TEMPLATE" }}
+</head>
+<body>
+	<div class="page-banner-image container-fluid d-flex position-relative justify-content-center align-items-center p-0">
```
Comment (foria) [inline src/docs/templates/our-story.hbs:8]: use SECTION

File: src/docs/templates/our-story.hbs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+				</div>
+			</div>
+			<figcaption class="col-12 col-lg-4">
+				<div class="image-with-text__text-content image-with-text__text-content--reverse text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pl-lg-3">
+					<h1>We've spent years testing, developing and perfecting our formulas</h1>
+					<p>We’re passionate about our customers, so we never use drying and damaging sulfates, phthalates or parabens, (unlike 98.9% of beauty products!). We also love our furry friends, so Coco & Eve is proudly 100% vegan and cruelty-free.</p>
+				</div>
+			</figcaption>
+	</div>
+	<div class="page-award-slider text-center pt-5 pb-5">
```
Comment (foria) [inline src/docs/templates/our-story.hbs:53]: use SECTION

File: src/docs/templates/our-story.hbs [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+		<div class="container ">
```
Comment (foria) [inline src/docs/templates/our-story.hbs:54]: we don't need to give the width to the P, also you are using a CONTAINER but without really using its features. So let's use the grid and col (maybe 3 cols size?) with centred col.

File: src/docs/templates/our-story.hbs [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+			<h2>We're #AwardWinning</h2>
+			<p class="ml-auto mr-auto mb-5">With 18 beauty awards under our belt, our products speak for themselves. Here’s a look at how we get down.</p>
+		</div>
+		<div id="carouselLoop4" class="carousel slide carousel--loop carousel--centered-mobile" data-slide-number="5">
```
Comment (foria) [inline src/docs/templates/our-story.hbs:58]: use SECTION

File: src/docs/templates/our-story.hbs [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+				<span class="sr-only">Previous</span>
+			</a>
+			<a class="carousel-control carousel-control-next carousel-control--background carousel-control--loop" href="#carouselLoop4" role="button" data-slide="next">
+				<span class="carousel-control-next-icon sni sni__chevron-next justify-content-center align-items-center" aria-hidden="true"></span>
+				<span class="sr-only">Next</span>
+			</a>
+		</div>
+	</div>
+
+	<div class="page-video-banner container-fluid justify-content-center d-lg-flex align-items-center p-0 position-relative">
```
Comment (foria) [inline src/docs/templates/our-story.hbs:96]: use SECTION, I don't think we need pos-rel

File: src/scss/components/_carousel.scss [hunk 7]
```diff
@@ -1,220 +1,242 @@
...
+			position: absolute;
+			top: 0;
+			left: 0;
+			z-index: -1;
+			display: block;
+			visibility: visible;
+		}
+
+		&[data-slide-number="5"] {
+			.col-md-3 {
```
Comment (foria) [inline src/scss/components/_carousel.scss:93]: if you want to use this just remove .col-md-3 from as class of the element and apply the rule on carousel-item

File: src/docs/templates/our-story.hbs [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+		<figure class="video-card video-card--play-caption ml-auto col-lg-7 p-0 position-relative mb-0">
+			<picture class=" sni sni__play video-card__icon-left video-card__icon-white position-relative" data-toggle="modal" data-src="https://www.youtube.com/embed/xwAfY_vdE9I" data-target="#videoCardModal">
+				<source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner05_DT_2x_eb45c1bd-6215-4dcf-8ded-c38f6931fe96_800x.jpg?v=1590061791" media="(min-width: 992px)">
+				<img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/AboutUs_banner05_DT_2x_eb45c1bd-6215-4dcf-8ded-c38f6931fe96_800x.jpg?v=1590061791" class="w-100 rounded-0">
+			</picture>
+			<figcaption>
```
Comment (foria) [inline src/docs/templates/our-story.hbs:102]: figcaption should be the text not this element, text can be a simple SPAN

File: src/scss/components/_video-card.scss [hunk 9]
```diff
@@ -21,6 +21,34 @@
...
 
+    &__icon-left {
+        &:before {
+            left: calc-em(20, 36);
+            right: auto;
+        }
+    }
+
+    &__icon-white {
+        &:before {
```
Comment (foria) [inline src/scss/components/_video-card.scss:32]: use same hierarchy used for the normal play, removing the !important.
&__icon-white {
picture {
        &:before {

**Review by foria — CHANGES_REQUESTED** [review]

File: src/scss/components/_video-card.scss (hunk 9 above)
Comment (teddymaulana) [reply src/scss/components/_video-card.scss]: not sure about this one, I just removed the icon white class , so its using normal play icon

**Pull request merged**